Client/index.css

```diff
@@ -0,0 +1,55 @@
+.main {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.logo {
+  position: relative;
+  width: 380px;
+  top: -50px;
+}
+
+.searchbar {
+  width: 500px;
+  height: 45px;
+  position: absolute;
+  transform: translate(-20%, -50%);
+}
+
```

Comment:
Your index page looks really nice. Very similar to google's

Reply:
I agree, I really like how you guys styled the homepage. I think it really great! For extra, I would add a navigation bar similar to google's and also a footer